recipes/power-grid-model-io/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "power-grid-model-io" %}
+{% set version = "1.2.37" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/power-grid-model-io-{{ version }}.tar.gz
+  sha256: 4c21060a961c831a506054b82aad4e19542dcaec3f5a327c15d7e05936f6ba65
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - numpy >=1.20
+    - openpyxl
+    - pandas
+    - power-grid-model >=1.4
+    - pyyaml
+    - structlog
+    - tqdm
+
+test:
+  imports:
+    - power_grid_model_io
```

Comment:
This `__init__` file is [empty](https://github.com/PowerGridModel/power-grid-model-io/blob/main/src/power_grid_model_io/__init__.py) so it wouldn't hurt having more import tests for other modules that do ship code. That said, since you are running `pytest` right now it's ok. I am only concerned that in the future the `pytest` part is removed and we only depend on a test that would almost always pass.

Reply:
Thanks for your reply. I have added a new import which is the core of all the essential functions.